WinUIGallery/Helper/WindowHelper.cs

```diff
@@ -65,8 +67,26 @@ static public Window GetWindowForElement(UIElement element)
             return null;
         }
 
-        static public List<Window> ActiveWindows { get { return _activeWindows; }}
+        static public List<Window> ActiveWindows { get { return _activeWindows; } }
 
         static private List<Window> _activeWindows = new List<Window>();
+
+        public static bool IsAppPackaged
+        {
+            get
+            {
+                int bufferSize = 0;
+                byte byteBuffer = 0;
+                // Try to grab the ID of the Package of the current process (if it exists)
```

Comment:
comment is inaccurate - grabbing the package ID is precisely what you're NOT doing here.